components/scripts/lib/build-scan-offline.sh

```diff
@@ -34,9 +34,20 @@ find_build_scan_dump() {
 }
 
 read_build_scan_dumps() {
-  local build_scan_csv
-  echo -n "Extracting build scan data"
-  build_scan_csv="$(invoke_java "$BUILD_SCAN_SUPPORT_TOOL_JAR" extract "0,${build_scan_dumps[0]}"  "1,${build_scan_dumps[1]}")"
+  local build_scan_csv args
+  args=()
+
+  args+=(
+      "extract"
+      "--license-file" "${SCRIPT_DIR}/gradle-enterprise.license"
+      "1,${build_scan_dumps[1]}"
+      "0,${build_scan_dumps[0]}"
+  )
+
+  echo "Extracting Build Scan data for all builds"
+  if ! build_scan_csv="$(invoke_java "$BUILD_SCAN_SUPPORT_TOOL_JAR" "${args[@]}")"; then
+    exit "$UNEXPECTED_ERROR"
```

Comment:
exit or die?

Reply:
Unfortunately we can't use `die` here because we don't have an error message (the offline tool takes care of printing the error messages).  We could call it, but then we would end up with an extra empty line after the scripts terminate.

In fact, I originally used `die` here, but switched it to `exit` when Eric pointed out the extra newline.